### ISSUE TITLE
CU7 Pageviewer Update - removed check

### ISSUE THREAD

**Issue by JasonBev** [LiveappSolutions/Liveapp#36]
Receiving errors across our development site after including CU7 update files. From basic data entry to inserting table keys. See image below- 

![image](https://user-images.githubusercontent.com/25164973/30260296-da112a10-9709-11e7-8963-a2ea02e7ba2a.png)

Traced to code changes below, as there is no check on if the variable 'f' exists before var field is defined, it throws an error on the undefined f.name.

![image](https://user-images.githubusercontent.com/25164973/30260275-b8039e80-9709-11e7-9ce6-8aa9d015cb73.png)

**Comment by paulfisher53** [comment]
I see the issue here and it's an easy fix. I'll get a fix to you shortly

**Comment by JasonBev** [comment]
Thanks for the update. I've just added a if(f) as a temp fix for now.

**Comment by paulfisher53** [comment]
This code fix will roll out with 4.2 CU9 (already fixed in v5):

```javascript
var f = m_record.GetField(filters[i][0]);
if(f){
  var field = m_table.Column(f.Name);                        
  if (f && field && filters[i][1] != f.Value && field.PrimaryKey){
    m_record.Filter(filters[i][0], f.Value);
  }
}
```

Note that `m_table.Column(f.name)` has been changed to `m_table.Column(f.Name)`
